Source/Core/Editor/UI/Drawers/ProcessSceneReferenceDrawer.cs

```diff
@@ -506,7 +540,7 @@ private void DrawSearchableGroupListPopup(Rect rect, Action<SceneObjectGroups.Sc
             UnityEditor.PopupWindow.Show(rect, content);
         }
 
-        private void UndoSceneObjectAutomaticSetup(GameObject selectedSceneObject, Type valueType, bool hadProcessComponent, Component[] alreadyAttachedProperties)
+        private void UndoSceneObjectAutomaticSetup(GameObject selectedSceneObject, Type valueType, Component[] alreadyAttachedProperties)
```

Comment:
Reverting a fixit will only remove the selected entety not any other entetys or components which where added.

This was an issue bevore, correct?
Do you know ist ther already a ticket for it?

Reply:
Yes I did not touch that function. Fix it undo is a bit broken and I'm pretty sure there's no ticket for it. I think it can be tricky to decide which components to remove when undoing.

Reply:
Why can we recorded wich components we added? May be I miss some edge cases.
I think doing a half undo is the worst as it gives the impression we do but actually leave a lot of garbage behind.
This behavior is not even documented.
Either we support it or not.

I'll create a ticket

Reply:
I agree, thanks for creating the ticket!